iree/compiler/Dialect/HAL/Target/LLVM/LLVMAOTTarget.cpp

```diff
@@ -76,6 +79,7 @@ class LLVMAOTTargetBackend final : public TargetBackend {
     passManager.addPass(createLowerExecutableTargetPass());
     // Set target specific options.
     // TODO(ataei): This is temporary here, should move when target specific
+
     // overrides options grows.
```

Comment:
revert added newline in the middle of this TODO

```suggestion
    // TODO(ataei): This is temporary here, should move when target specific
    // overrides options grows.
```